kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -290,6 +290,11 @@ form .input-group {
     background-color: #40775F !important;
 }
 
+.login-btn, btn-block {
+    display: block;
+    width: 100%;
```

Comment:
Can you set SIMPLIFIED_LOGIN=True in your local_settings.py and see how this width setting interacts with the additional button that appears with this setting?